karma.conf.js

```diff
@@ -27,23 +27,43 @@ module.exports = function(config) {
 
       'jujugui/static/gui/build/app/assets/javascripts/handlebars.runtime.js',
 
-      'jujugui/static/gui/build/app/components/**/*.js',
-      'jujugui/static/gui/build/**/test-*.js'
+      'jujugui/static/gui/src/app/components/**/*.hbs',
+      'jujugui/static/gui/src/app/components/**/*.js'
     ],
 
 
     // list of files to exclude
     exclude: [
-      'jujugui/static/gui/build/app/components/**/*-min.js',
-      'jujugui/static/gui/build/**/test-*-min.js',
```

Comment:
This line plus the two below seemed like cruft to me, so I removed/replaced them with the single line. If they are, in fact, useful, then let me know.